apps/prairielearn/src/lib/db-types.ts

```diff
@@ -101,3 +102,51 @@ export const UserSchema = z.object({
   user_id: IdSchema,
 });
 export type User = z.infer<typeof UserSchema>;
+
+export const QuestionSchema = z.object({
```

Comment:
Yeah, we could, but then we're writing our own ORM 😅. I haven't found it a huge burden in PT to maintain this by hand, but it would be nice to make sure that the nullable status was set correctly, for example. I'm not sure this is a super high priority, though.